assets/queries/k8s/app_armor_config/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "app_armor_config",
+  "queryName": "app_armor_config",
```

Comment:
The query name and id should explain the vulnerability. Try something like "Missing_App_Armor_Config"

Reply:
You are correct Ruben! I'll change it per suggestion